x-pack/plugins/lens/public/xy_visualization_plugin/xy_visualization.tsx

```diff
@@ -6,19 +6,36 @@
 
 import React from 'react';
 import { render } from 'react-dom';
-import { Visualization, DimensionRole } from '../types';
-
-export interface XyVisualizationState {
-  roles: DimensionRole[];
-}
-
-export type XyVisualizationPersistedState = XyVisualizationState;
-
-export const xyVisualization: Visualization<XyVisualizationState, XyVisualizationPersistedState> = {
-  initialize() {
-    return {
-      roles: [],
-    };
+import { Position } from '@elastic/charts';
+import { Visualization } from '../types';
+import { getSuggestions } from './xy_suggestions';
+import { XYArgs } from './xy_expression';
+
+export const xyVisualization: Visualization<XYArgs, XYArgs> = {
```

Comment:
Suggestion: For now XYArgs is the same as the state of the visualization, but it might not stay like this. What about having a type alias here which is used for suggestion building etc.? `type XYState = XYArgs;`

Reply:
I'm OK with an alias. My thought was that we'd create a specific state type when the time comes, and only if we need it, rather than starting out with it. But maybe an alias is the right course of action?

Reply:
I like an alias because if we have it in there from the start it's very easy to add more stuff the state later. When we don't have it, we have to go through all the files and figure out whether it's the args or the state. Not super important though, I was just confused by a `VisualizationSuggestion<XYArgs>`